bcos-executor/src/executive/ExecutiveFactory.cpp

```diff
@@ -36,7 +37,7 @@ using namespace bcos::precompiled;
 
 
 std::shared_ptr<TransactionExecutive> ExecutiveFactory::build(
-    const std::string& _contractAddress, int64_t contextID, int64_t seq, bool useCoroutine)
+    const std::string& _contractAddress, int64_t contextID, int64_t seq, bool useCoroutine, bool isSerialFlow)
```

Comment:
```suggestion
    const std::string& _contractAddress, int64_t contextID, int64_t seq, int executiveType)
```

type
0: normal
1: useCoroutine
2: billing